### ISSUE TITLE
Vagrant up error (not first time load)

### ISSUE THREAD

**Issue by joelworsham** [Varying-Vagrant-Vagrants/VVV#1088]
## Expected Behavior
Vagrant should start up without any issues.

## Current Behavior
Error shows on startup:

`==> default: /home/vagrant/.bash_profile: line 39: /srv/config/wp-cli/wp-completion.bash: No such file or directory`

## Possible Solution
Well, in the `bash_profile` file in `config/`, it is calling a file within `wp-cli/` that simply doesn't exist. I checked and it does not exist inside this repo at all.

## Steps to Reproduce (for bugs)
1. Startup Vagrant

## Your Environment
* VVV version: develop
* Vagrant version: 1.7.4
* VM Provider name: VirtualBox
* VM Provider version: 4.3.36
* Operating System and version: Mac - 10.11.6

**Comment by joelworsham** [comment]
UPDATE: Updated Vagrant and VirtualBox to latest, same issue. Also, lots of DB backup issues when halting. `No such file or directory` on every backup.sql file.

**Comment by LoreleiAurora** [comment]
Can you pull the latest develop and reprovision?

**Comment by joelworsham** [comment]
I've already tried that, good idea though. 

Also, through re-provisioning many times, manually deleting then provisioning things, and also manually uninstalling wp-cli and provisioning, I've alleviated that error it seems.

I do, however, have another issue... All of the default domains (src.wordpress-develop.dev, etc) go to the main vvv.dev screen. I've even tried manually clearing the hosts file and then reloading vagrant, nothing works.
